Services/source/updater/updater.c

```diff
@@ -12,14 +12,33 @@
 #include "bl_flash.h"
 #include "privileged_functions.h"
 
+//for testing only. do hex dump
+//size is the number of bytes we want to print
+static void hex_dump(char *stuff, int size){
```

Comment:
Maybe this could be moved to a more visible place (like `services.c`), since it might be useful elsewhere

Reply:
I think a definition of this is still in housekeeping (hence the static) so it would need to be removed there